cmd/tlssocksproxy/tlssocksproxy.go

```diff
@@ -6,166 +6,144 @@ import (
 	"flag"
 	"io"
 	"net"
-	"net/http"
+	"sync/atomic"
 	"time"
 
-	"github.com/foomo/tlssocks"
-	"github.com/pkg/errors"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/foomo/tlssocks/cmd"
 	"go.uber.org/zap"
-	"golang.org/x/sync/errgroup"
-)
-
-var (
-	logger *zap.Logger
 )
 
 const (
 	defaultTimeout           = 180 * time.Second
 	defaultPrometheusAddress = ":9200"
+	connDeadline             = 60 * time.Second
```

Comment:
AKA time.Minute :)

Reply:
I'm not sure about the correct timeouts in all cases ...